sample_app/main_rtk.c

```diff
@@ -286,7 +286,11 @@ int main(void)
 
          os_event_clr(appdata.main_events, EVENT_ALARM); /* Re-arm */
 
-         ret = pnet_alarm_send_ack(g_net, appdata.main_arep, &pnio_status);
+         ret = pnet_alarm_send_ack(g_net, appdata.main_arep,
```

Comment:
That's quite a lot of arguments. Would it make sense to introduce an alarm type (consisting of slot, subslot, type, sequence_number and specifier)?

Reply:
Yes, I can do that.

Reply:
Implemented an alarm argument type